src/components/Manifesto.tsx

```diff
@@ -2,111 +2,85 @@ import React from "react";
 import styled from "styled-components";
 import { useTranslation } from "react-i18next";
 import {
-  color,
   layout,
   space,
   typography,
-  background,
   flexbox,
-  border,
   grid,
-  ColorProps,
   LayoutProps,
   SpaceProps,
   TypographyProps,
-  BackgroundProps,
   FlexboxProps,
-  BorderProps,
   GridProps,
 } from "styled-system";
+import Header from "./Header";
 
-const Main = styled.main<
-  ColorProps & SpaceProps & BackgroundProps & LayoutProps & FlexboxProps
->`
+const Main = styled.main<SpaceProps & FlexboxProps>`
   min-height: 100vh;
-  overflow: hidden;
-  ${color};
+  display: flex;
   ${space};
-  ${background};
-  ${layout};
   ${flexbox}
 `;
 
-const H1 = styled.h1<ColorProps & TypographyProps & SpaceProps>`
-  text-transform: uppercase;
-  ${color};
-  ${typography};
+const Container = styled.div<SpaceProps & LayoutProps>`
   ${space};
+  ${layout};
 `;
 
-const Grid = styled.div<GridProps & FlexboxProps & SpaceProps>`
+const Grid = styled.div<GridProps & FlexboxProps>`
   display: grid;
   ${grid};
   ${flexbox};
-  ${space};
 `;
 
-const Container = styled.div<
-  ColorProps & SpaceProps & BorderProps & LayoutProps
->`
-  ${color};
+const H1 = styled.h1<TypographyProps & SpaceProps & GridProps>`
+  text-transform: uppercase;
+  ${typography};
   ${space};
-  ${border}
-  ${layout};
+  ${grid};
 `;
 
-const Paragraph = styled.p<ColorProps & TypographyProps & SpaceProps>`
+const Paragraph = styled.p<TypographyProps & SpaceProps>`
   text-transform: uppercase;
-  ${color};
   ${typography};
   ${space};
+  padding-bottom: 20px;
```

Comment:
this can be a prop on the Component since you're already using `space`

Reply:
It applies to all <P> tags though so I'm trying to keep it as DRY as poss i.e. say it once in the Component and void `pb`ing everywhere

Reply:
sure, lets leave it like this but its good to be aware that if (in the unlikely case) the theme changed then this padding-bottom wouldnt change

Reply:
what's best practice? to use props in every case?

what if, for example, an element only uses one prop once like background color? (i know we're using Global Styles so probs bad example) do you pass the prop which is essentially more code or do you just go with old-fashioned CSS